packages/app/src/components/Admin/Customize/CustomizeLogoSetting.tsx

```diff
@@ -0,0 +1,166 @@
+import React, { FC, useState } from 'react';
+
+import { useTranslation } from 'react-i18next';
+
+
+import AdminCustomizeContainer from '~/client/services/AdminCustomizeContainer';
+import AppContainer from '~/client/services/AppContainer';
+import {
+  toastError, toastSuccess,
+} from '~/client/util/apiNotification';
+import ImageCropModal from '~/components/Common/ImageCropModal';
+
+import { withUnstatedContainers } from '../../UnstatedUtils';
+import AdminUpdateButtonRow from '../Common/AdminUpdateButtonRow';
+
+
+type Props = {
+  adminCustomizeContainer : AdminCustomizeContainer
+}
+
+const CustomizeLogoSetting: FC<Props> = (props: Props) => {
+
+  const { t } = useTranslation();
+  const { adminCustomizeContainer } = props;
+  const [isShow, setIsShow] = useState<boolean>(false);
+  const [src, setSrc] = useState<ArrayBuffer | string | null>(null);
+  const {
+    uploadedLogoSrc, isDefaultLogo, defaultLogoSrc,
+  } = adminCustomizeContainer.state;
+
```

Comment:
Adding `showModal` would be more readable if there are `hideModal` & `cancelModal` that are delegated to change state of `setIsShow`.

like this
```
  const showModal = () => {
    setIsShow(true);
  };
```

Reply:
Added `showModal` method